mission1/index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta property="og:title" content="판다 마켓" />
+    <meta property="og:description" content="일상의 모든 물건 을 거래해보세요" />
+    <meta property="og:url" content="/" />
+    <link rel="stylesheet" href="style.css" />
+    <title>판다마켓</title>
+  </head>
+  <body>
+    <div id="wrap">
+      <!-- HEADER -->
+      <div class="header_wrap">
+        <header class="header">

```

Comment:
💊 제안
아래처럼 두 태그의 순서가 바뀌는 것이 더 파악하기 쉬울 것 같아요!

```suggestion
      <header class="header">
        <div class="header_inner">
```

이렇게 되면 header 컴포넌트의 최상단 컴포넌트가 header이므로 영역파악도 좀 더 쉽고, 주석을 통해 구분하실 필요도 없습니다~